.github/workflows/static_checks.yml

```diff
@@ -10,6 +10,8 @@ jobs:
 
       - name: Install dependencies
         run: |
+          echo '::add-path::$HOME/.local/bin'
+          export PATH=$HOME/.local/bin:$PATH # ::add-path doesn't work for the current action.
```

Comment:
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#adding-a-system-path says it doesn't work for the current "job", but if you don't need it in the "Install dependencies" job, we can skip this. It should be available for the black job further down.

Reply:
pip3 complains about it, but I suppose we could silence the warning instead.

Reply:
Or move it to an earlier job.

Reply:
We could also install system-wide with pip and not both with all this :)

Reply:
i.e. just `s/ --user//`.

Reply:
Damn, it seems `echo '::add-path::$HOME/.local/bin'` is not working. Black still fails...

I would like to find a solution that avoids hard-coding `/home/runner`, so now I'm thinking calling `$HOME/.local/bin/black` in the black script is the best option that we have at the moment.

Reply:
Keep it simple:
```diff
diff --git a/.github/workflows/static_checks.yml b/.github/workflows/static_checks.yml
index bcba4c36d9..587f5a6ba6 100644
--- a/.github/workflows/static_checks.yml
+++ b/.github/workflows/static_checks.yml
@@ -12,7 +12,7 @@ jobs:
         run: |
           sudo apt-get update -qq
           sudo apt-get install -qq dos2unix recode clang-format
-          pip3 install --user black pygments
+          sudo pip3 install black pygments
 
       - name: File formatting checks (file_format.sh)
         run: |
```

Reply:
Yes, that's a good idea. I blame [whoever wrote the line I copied](https://github.com/godotengine/godot/commit/3644036fd3d8678ac44695cc49ae63c4aaeb1b97#diff-354f30a63fb0907d4ad57269548329e3R152). Seems the culprit is @akien-mga :P

Reply:
Well on Travis we were using the sudo-less mode, which gives better containers. :)